docs/csharp/language-reference/statements/fixed.md

```diff
@@ -9,7 +9,7 @@ helpviewer_keywords:
   - "fixed statement [C#]"
   - "fixed keyword [C#]"
 ---
-# fixed statement - safely access memory underlying a variable
+# fixed statement - pin a variable for pointer operations
```

Comment:
Original text may give an impression that by default, without `fixed`, you cannot safely access variable. In the new version, I also would like to point that `fixed` is for scenarios with pointers, so most probably, a rare use.

Reply:
I like it. That's a great edit.